lerna.json

```diff
@@ -4,5 +4,5 @@
   ],
   "npmClient": "yarn",
   "useWorkspaces": true,
-  "version": "1.5.0-alpha.1"
+  "version": "1.5.0-alpha.2.filters.1"
```

Comment:
@Josmorsot We don't usually use custom name alphas for c4r, as we do in kepler. So I would recommend leaving current in the PR as the first step, to merge in master, and then make another PR for the new alpha release (1.5.0-alpha.2)